src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+const DAILY_RENTAL_RATE = 40;
+const LONG_TERM = 7;
+const SHORT_TERM = 3;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM_DISCOUNT = 20;
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < SHORT_TERM) {
+    return days * DAILY_RENTAL_RATE;
+  } else if (days < LONG_TERM) {
+    return days * DAILY_RENTAL_RATE - SHORT_TERM_DISCOUNT;
+  }
+
+  return days * DAILY_RENTAL_RATE - LONG_TERM_DISCOUNT;
```

Comment:
```suggestion
  if (days < SHORT_TERM) {
    return days * DAILY_RENTAL_RATE;
  } 
  
  if (days < LONG_TERM) {
    return days * DAILY_RENTAL_RATE - SHORT_TERM_DISCOUNT;
  }

  return days * DAILY_RENTAL_RATE - LONG_TERM_DISCOUNT;
```

<img width="1100" height="382" alt="Screenshot 2025-07-23 at 11 39 51" src="https://github.com/user-attachments/assets/46acb1bc-c5a2-4f9a-afe0-17b71a09c805" />